apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktService.kt

```diff
@@ -392,6 +416,26 @@ class AktivitetspliktService(
             }
     }
 
+    private fun opprettOppgaveForVarigUnntak(
+        request: OpprettOppgaveForAktivitetspliktVarigUnntakDto,
+    ): OpprettOppgaveForAktivitetspliktVarigUnntakResponse {
+        logger.info("Oppretter oppgave for infobrev for varig unntak av aktivitetsplikt for sak ${request.sakId}")
+        return oppgaveService
+            .opprettOppgave(
+                sakId = request.sakId,
+                referanse = request.referanse ?: "",
+                kilde = OppgaveKilde.HENDELSE,
```

Comment:
Blir det riktig med hendelse her?

Reply:
Vil tro det skal være hendelse siden dette trigges 6mnd etter dødsfallsdato?

hendelse er også brukt for opprettelse av revurdering og oppgave for aktivitetsplikt ved 4mnd